library/objective-c/Envoy.h

```diff
@@ -10,5 +10,10 @@
 
 /// Create a new Envoy instance. The Envoy runner NSThread is started as part of instance
 /// initialization with the configuration provided.
-- (instancetype)initWithConfig:(NSString*)config;
+- (instancetype)initWithConfig:(NSString *)config;
+
+/// Create a new Envoy instance. The Envoy runner NSThread is started as part of instance
+/// initialization with the configuration provided.
+- (instancetype)initWithConfig:(NSString *)config logLevel:(NSString *)logLevel;
```

Comment:
Would also be nice to document what valid log levels are available here

Reply:
I agree enums would be better. Tell you all what I'll open another PR with enums after I merge and re-test iOS. :)